torch_xla/csrc/tensor.cpp

```diff
@@ -1840,6 +1840,11 @@ c10::SymIntNode XLASymIntNodeImpl::mul(const c10::SymIntNode& other) {
   return c10::make_intrusive<XLASymIntNodeImpl>(nmul);
 }
 
+c10::SymIntNode XLASymIntNodeImpl::wrap(int64_t num) {
```

Comment:
What's the expected behavior for "wrap"? I couldn't find the documentation in the design doc.

Reply:
I think you also reviewed `SizeConstant` PR as well. We need `SizeConstants` or constants since we may have constants in size expressions. For example, `x.shape[0] * 2` or `x.shape[0] / 2`.
The easiest way to make it work is to use the other argument of "+" x.shape[0] to turn 2 into `SizeConstant(2)` since the other argument is already a proper `SymInt`. This is why we have `wrap`

Reply:
I see. So `wrap` does a cast. Thanks for explaining.